app/views/admin/base/dashboard.html.erb

```diff
@@ -5,5 +5,6 @@
 
 <p><%= link_to "Categories", admin_categories_path %></p>
 <p><%= link_to "Tenants", admin_tenants_path %></p>
```

Comment:
I thought we took this out already? I do prefer "borrowers" to "tenants". Personally, if these are the only three links, I don't see anything wrong with having them all in the navbar...